src/styles/blocks/form.css

```
@@ -0,0 +1,36 @@
+.form {
+  padding-top: 40px;
+}
+
+.form__wrapper {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+}
+
+.form__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 36px;
+  grid-column: span 10 / span 10;
+  grid-column-start: 2;
```

в фигме видно что отступ в 2 колонки, поэтому тут все проще
```suggestion
    grid-column-start: 3;
    grid-column-end: 11;
```